tests/System.Buffers.Primitives.Tests/CopyTo.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Xunit;
+
+namespace System.Buffers.Tests
+{
+    public class CopyTo
+    {
+        [Fact]
+        public static void CopyToSpan()
+        {
+            int[] src = { 1, 2, 3 };
+            int[] dst = { 99, 100, 101 };
+
+            var srcBuffer = new Buffer<int>(src);
```

Comment:
There is already coverage for implicit cast in the equality tests in https://github.com/dotnet/corefxlab/pull/1607

I think a better place to make sure we have coverage for the implicit cast is to add some tests to CtorArray.cs and CtorArrayReadOnlyBuffer.cs. Thoughts?